### PR TITLE
issue #636  @censor_word_detection

### LINKED ISSUE #636: Censor word detection
I am GSSOC 21 participant. And I would like to contribute here by making program that can detect censor word

### PR DESCRIPTION
**Pull request by jeelnathani**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #636

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [ ] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [ ] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Censor word detection/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+</ul>
+
+<h3> Approach </h3>
+<ol>
+  <li>Import the module profanity from better_profanity</li>
+  <li>Take user input </li>
+  <li>Check for the censored data using method profanity.censor() </li>
+  <li>Print the output on terminal. 
+    <ul>
+      <li>It will print **** if profanity.censor(text) is used.</li>
```
Comment (kaustubhgupta) [inline Censor word detection/README.md:21]: this never printed?

File: Censor word detection/censor_word_detection.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from better_profanity import profanity
+
+def detect(data):
+    try:
+        '''profanity.censor(data) will print **** '''
```
Comment (kaustubhgupta) [inline Censor word detection/censor_word_detection.py:5]: this line is not need as it never printed ***

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by jeelnathani** [comment]
changes are done

File: Censor word detection/censor_word_detection.py (hunk 2 above)
Comment (kaustubhgupta) [reply Censor word detection/censor_word_detection.py:5]: @jeelnathani this is still left

**Comment by jeelnathani** [comment]
Done

**Review by kaustubhgupta — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]
LGTM 💯

**Review by antrikshmisri — APPROVED** [review]
LGTM 👍🏻

**Review by avinashkranjan — APPROVED** [review]
LGTM..🚩

**Pull request merged**